deploy/common/k8sbased.sh

```diff
@@ -583,17 +591,28 @@ function launch_sensor {
         )
       fi
 
+      local helm_chart="$k8s_dir/sensor-deploy/chart"
```

Comment:
Similarly,

```suggestion
      local helm_chart="${SENSOR_CHART_DIR_OVERRIDE:-${k8s_dir}/sensor-deploy/chart}"
```

plus (***)